starters/servers/cloudflare/wrangler.toml

```diff
@@ -0,0 +1,18 @@
+name = "qwik-app"
```

Comment:
`qwik-project-name` right?

Reply:
yes!

Reply:
need to modify the CLI to not copy... but read and write. cuz I don't have a way to find replace
in other files other than package.json